### PR TITLE
Tweak passwordless and device trust testplan

### PR DESCRIPTION
**Pull request by codingllama**
Do a few small changes and additions to the passwordless and device trust sections.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/testplan.md [hunk 1]
```diff
@@ -506,7 +506,9 @@ instance has label `azure/foo=bar`.
 
 ### Passwordless
 
-This feature has additional build requirements, so it should be tested with a pre-release build from Drone (eg: `https://get.gravitational.com/teleport-v10.0.0-alpha.2-linux-amd64-bin.tar.gz`).
+This feature has additional build requirements, so it should be tested with a
+pre-release build from Drone (eg:
+`https://get.gravitational.com/tsh-v10.0.0-alpha.2.pkg`).
```
Comment (codingllama) [inline .github/ISSUE_TEMPLATE/testplan.md:511]: I think this makes more sense, as the macOS tsh is actually the "difficult" binary, others don't need anything special to work.

File: .github/ISSUE_TEMPLATE/testplan.md [hunk 2]
```diff
@@ -568,7 +570,7 @@ Device Trust requires Teleport Enterprise.
 
 This feature has additional build requirements, so it should be tested with a
 pre-release build from Drone (eg:
-`https://get.gravitational.com/teleport-v10.0.0-alpha.2-linux-amd64-bin.tar.gz`).
+`https://get.gravitational.com/teleport-ent-v10.0.0-alpha.2-linux-amd64-bin.tar.gz`).
```
Comment (codingllama) [inline .github/ISSUE_TEMPLATE/testplan.md:573]: Similarly here, but we actually want the Enterprise build in this case.

**Comment by codingllama** [comment]
New scenarios already covered for v14.

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 1 above)
Comment (sshahcodes) [reply .github/ISSUE_TEMPLATE/testplan.md:511]: good idea. maybe also update the version to `v14` while we are at it?

File: .github/ISSUE_TEMPLATE/testplan.md [hunk 3]
```diff
@@ -667,6 +670,12 @@ tsh ssh node-that-requires-device-trust
   - [ ] Non-signed and/or non-notarized `tsh` for macOS gives a sane error
         message for `tsh device enroll` attempts.
 
+- [ ] Device support commands
+  - [ ] `tsh device collect`   (macOS)
+  - [ ] `tsh device asset-tag` (macOS)
```
Comment (sshahcodes) [inline .github/ISSUE_TEMPLATE/testplan.md:675]: Not related to this PR, but looks like we can deprecate `tsh device collect` in favor of `tsh device asset-tag`?

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 2 above)
Comment (sshahcodes) [reply .github/ISSUE_TEMPLATE/testplan.md:573]: update reference version here as well?

**Review by flyinghermit — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 1 above)
Comment (codingllama) [reply .github/ISSUE_TEMPLATE/testplan.md:511]: I don't think it's worth it. By the time anyone reads this copy, the version will have changed again.

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 3 above)
Comment (codingllama) [reply .github/ISSUE_TEMPLATE/testplan.md:675]: `tsh device collect` is useful for debugging, I'd rather keep it.

**Review by avatus — APPROVED** [review]

**Pull request merged**